setup.py

```diff
@@ -54,6 +54,7 @@
         "grpcio >= 1.47.0, < 2.0dev",
         "grpcio >= 1.49.1, < 2.0dev; python_version>='3.11'",
     ],
+    "bigframes": ["bigframes >= 1.17.0"],
```

Comment:
`pip` will always install the latest version of bigframes. Let's make sure we test our compatibility with the minimum supported bigframes by adding `bigframes==1.17.0` to https://github.com/googleapis/python-bigquery-magics/blob/main/testing/constraints-3.9.txt

Reply:
Done